src/components/table.js

```diff
@@ -187,13 +187,17 @@ const ariaSort = (sort, field) => {
   return null
 }
 
+const noContent = ({ noContentMessage, noContentRenderer = _.noop, defaultMessage = 'Nothing to display' }) => {
```

Comment:
Since `defaultMessage` isn't overridden anywhere, can we make it not be an argument?

Reply:
The idea was that different things that use this function could potentially have different defaults. But you're right, I'm not currently using that, and don't have any plans to. I'll strip it out.